components/org.wso2.carbon.identity.governance/src/main/java/org/wso2/carbon/identity/governance/IdentityGovernanceServiceImpl.java

```diff
@@ -57,6 +58,11 @@ public void updateConfiguration(String tenantDomain, Map<String, String> configu
                 prop.setName(key);
                 if (configurationDetails.containsKey(key)) {
                     prop.setValue(configurationDetails.get(key));
+                    if (Objects.equals(key, "EmailOTP.OtpRegex.Migrated")) {
```

Comment:
What is the meaning of this key?

Reply:
This is the hidden config that we maintain to track if the config value of useNumeric if migrated to useAlphaNumeric config.